clients/integrations/gymnasium/tests/unit/test_gymnasium_env.py

```
@@ -0,0 +1,210 @@
+    env = SandboxEnv(
+        reward_fn=reward_fn,
+        termination_fn=termination_fn,
+        client=mock_client
+    )
+
+    env.reset()
+
+    # Simulate an exception during run
+    mock_sandbox.commands.run.side_effect = Exception("Connection lost")
+
+    obs, reward, terminated, truncated, info = env.step("echo hello")
```

Ruff (0.16.1)

[warning] Unpacked variable `reward` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)